Seravo/seravo-plugin
# Fix dependency loading and update svg-sanitizer

#### What are the main changes in this PR?

- Fixes 3rd party dependency loading.
- Updates SVG-Sanitizer.

#### Manual testing steps?
- Upload `.svg` file on media page (`/wp-admin/upload.php`).
- Remove global svg-sanitizer if one exists (`rm -rf /data/wordpress/vendor/enshrined/svg-sanitize/`).
  - Might cause errors on some new sites, report here if it does.
- Upload `.svg` file again and see it still works.
